### PR TITLE
Updating SUMMARY files for PDF generation

### PR DESCRIPTION
**Pull request by stuclem**
Updated individual SUMMARY files for the PDFs, to pull in latest changes

### CODE REVIEW

File: docs/user_doc/vic_cloud_admin/SUMMARY.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
 <!-- USE THIS SUMMARY FILE FOR PDF GENERATION ONLY -->
 
 * [Configure and Manage](README.md)
+   * [Logging in to the Management Portal](logging_in_mp.md)
```
Comment (anchal-agrawal) [inline docs/user_doc/vic_cloud_admin/SUMMARY.md]: This doesn't resolve - I think it should be `logging_in_MP.md`

Comment (stuclem) [reply docs/user_doc/vic_cloud_admin/SUMMARY.md]: It works in the PDF and HTML output:
https://stuclem.github.io/vic-product/assets/files/html/1.2/vic_cloud_admin/logging_in_mp.html
https://stuclem.github.io/vic-product/assets/files/html/1.2/pdf/vic_12_cloud_admin.pdf (page 4)

Comment (anchal-agrawal) [reply docs/user_doc/vic_cloud_admin/SUMMARY.md]: Does changing it to `logging_in_MP.md` break the PDF and HTML renders?

Comment (stuclem) [reply docs/user_doc/vic_cloud_admin/SUMMARY.md]: No, it doesn't.

**Pull request merged**